modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/UpNextAdapter.kt

```diff
@@ -166,9 +171,32 @@ class UpNextAdapter(
                     root.resources.getQuantityString(LR.plurals.player_up_next_header_title, header.episodeCount, header.episodeCount, time)
                 }
 
+                shuffle.isVisible = FeatureFlag.isEnabled(Feature.UP_NEXT_SHUFFLE)
+                shuffle.setShuffleDisabledButton(context)
+
+                shuffle.setOnClickListener {
+                    shuffle.setShuffleEnabledButton(context)
+                }
+
                 root.layoutParams.height = ViewGroup.LayoutParams.WRAP_CONTENT
             }
         }
+
+        private fun ImageButton.setShuffleDisabledButton(
+            context: Context,
+        ) {
+            this.setImageDrawable(ContextCompat.getDrawable(context, IR.drawable.shuffle))
+            this.contentDescription = context.getString(LR.string.up_next_shuffle_button_content_description)
+            this.setImageTintList(ColorStateList.valueOf(ThemeColor.primaryIcon02(theme)))
```

Comment:
```
TooltipCompat.setTooltipText(this, context.getString(LR.string.up_next_shuffle))
```
I wonder if it would be a good idea to have a tooltip similar to how you long press icons in the Toolbar.

![Screenshot_20241022_160501](https://github.com/user-attachments/assets/0c12cdcd-580a-46ca-8f35-1268138c9f24)

Reply:
I liked that! I will add it